### PR TITLE
fix(connectors-qa): reject default airbyte icons in connectors

### PR DESCRIPTION
**Pull request by natikgadzhi**
## What

Adds a test in `connectors-qa` `CheckConnectorIconIsAvailable` to ensure connector has a non-default icon.

/cc @marcosmarxm 

This PR should be merged before #46746 gets in, so I'd like to get this in quickly.

### CODE REVIEW

**Review by clnoll — APPROVED** [review]
Approving because it shouldn't break anything, but I don't think I have the context to know that it will for sure fix the problem.

File: airbyte-ci/connectors/connectors_qa/README.md [hunk 1]
```diff
@@ -108,14 +108,18 @@ poe lint
 
 ## Changelog
 
+### 1.18.1
```
Comment (natikgadzhi) [inline airbyte-ci/connectors/connectors_qa/README.md:111]:
```suggestion
### 1.19.1
```

File: airbyte-ci/connectors/connectors_qa/src/connectors_qa/checks/assets.py [hunk 2]
```diff
@@ -19,14 +22,21 @@ def _run(self, connector: Connector) -> CheckResult:
...
-                message="Icon file is missing. Please create an icon file at the root of the connector code directory.",
+                message="Icon file is missing. Please create an icon file at the root of the connector code directory",
             )
         if not connector.icon_path.name == "icon.svg":
             return self.create_check_result(
                 connector=connector,
                 passed=False,
                 message="Icon file is not named 'icon.svg'",
             )
+        if connector.icon_path.read_text() == DEFAULT_AIRBYTE_ICON:
```
Comment (alafanechere) [inline airbyte-ci/connectors/connectors_qa/src/connectors_qa/checks/assets.py:33]:
```suggestion
        if connector.icon_path.read_text().strip().lower() == DEFAULT_AIRBYTE_ICON.strip().lower():
```

Comment (alafanechere) [reply airbyte-ci/connectors/connectors_qa/src/connectors_qa/checks/assets.py:33]: I think we should handle exception that `read_text` might raise if the file is not text.

**Review by alafanechere — COMMENTED** [review]
I'll push the commented improvements

**Review by alafanechere — APPROVED** [review]
I improved SVG validation, fixed unit tests and added a couple.

**Comment by natikgadzhi** [comment]
Damn type checks. @alafanechere get this pushed in and merged if you'll have a second, please. I might get to it Wednesday afternoon.

**Pull request merged**